flyway-commandline/src/main/assembly/component.xml

```diff
@@ -193,7 +193,13 @@
             <useTransitiveFiltering>true</useTransitiveFiltering>
             <includes>
                 <include>com.microsoft.azure:msal4j</include>
+                <include>com.azure:azure-identity</include>
```

Comment:
@piers-williams , before creating the issue I did a clean test with a clean zip download from [GH releases](https://github.com/flyway/flyway/releases/tag/flyway-11.0.0) page.
Also, as I was fixing it, I could confirm that it works the same when packaged myself from the `main` branch.
Also, after the fix, I could confirm that it helped (actually, the list of missing deps was a back and forth process until it started working).

As per your request I downloaded release from the page you kindly provided a link for and imagine my surprise when I found out that its size is 423 MiB vs 222 MiB that's available at GH releases.

As for the tests:
1. [GH release](https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/11.0.0/flyway-commandline-11.0.0-windows-x64.zip) - issue is reproducible.
2. [Download page release](https://download.red-gate.com/maven/release/com/redgate/flyway/flyway-commandline/11.0.0/flyway-commandline-11.0.0-windows-x64.zip) - not reproducible.
3. ZIP built from the clean source code - issue is reproducible
4. ZIP built from the PR source code - resolves the issue
5. ZIP downloaded from [maven](https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/11.0.0/) - issue is reproducible

It looks like zip from the download page is not shipped from the source code in this repo :).

It affects us since we use [Azure DevOps Extension](https://github.com/mfilotto/azure-pipelines-tasks-flyway/blob/master/FlywayInstaller/src/installer.ts) that downloads Flyway from the maven releases page.

Reply:
Also see https://github.com/flyway/flyway-docker/issues/171